### PR TITLE
[PyROOT] Sanitize input arguments to AsNumpy

### PR DESCRIPTION
**Pull request by stwunsch**
AsNumpy('foo') would iterate over the characters of 'foo'. Notify the
user that we expects AsNumpy(['foo']) in the case a plain string is
passed.

### CODE REVIEW

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py [hunk 1]
```diff
@@ -49,6 +49,12 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     Returns:
         dict: Dict with column names as keys and 1D numpy arrays with content as values
     """
+    # Sanitize input arguments
+    if not columns == None and isinstance(columns, str):
```
Comment (etejedor) [inline bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py:53]: Do you strictly need to compare with `None` first?

Comment (stwunsch) [reply bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py:53]: uhm no, thanks :see_no_evil:

Comment (stwunsch) [reply bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py]: Fixed!

**Review by etejedor — APPROVED** [review]
Thanks @stwunsch !

File: bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py [hunk 2]
```diff
@@ -49,6 +49,12 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     Returns:
         dict: Dict with column names as keys and 1D numpy arrays with content as values
     """
+    # Sanitize input arguments
+    if isinstance(columns, str):
```
Comment (vepadulano) [inline bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py:53]:
```suggestion
    if not isinstance(columns, list):
```
If we want to check that the user passes a list I'd say this is better.

Comment (stwunsch) [reply bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py:53]: I thought about this but in python a lot of objects can be iterable. Also, a list is not a tuple, just for example see here:

```python
>>> isinstance([], list)
True
>>> isinstance((), list)
False
```

**Pull request merged**